includes/tracks/class-wc-tracks.php

```diff
@@ -33,13 +37,23 @@ public static function get_products_count() {
 	 */
 	public static function get_blog_details( $user_id ) {
 		$blog_details = get_transient( 'wc_tracks_blog_details' );
+
+		$product_counts = WC_Tracker::get_product_counts();
```

Comment:
I guess this does not need to run if the `$blog_details` are cached. I'd put it inside the if below.